components/VotesList/VotesListItem.tsx

```diff
@@ -83,23 +83,22 @@ export function VotesListItem({
   const isRolled = rollCount > 0;
   const wrapperRef = useRef<HTMLTableRowElement>(null);
   const existingVote = getDecryptedVoteAsFormattedString();
-  useEffect(() => {
-    if (!options) return;
```

Comment:
I think this is from old code — there's no need to do this check. and having an object like this in the dependency array causes this hook to run on every render, which means if you try to change the input back from custom it will get changed back to custom again on every render.